paparazzi-gradle-plugin/src/test/projects/lifecycle-usages/build.gradle

```diff
@@ -20,6 +20,6 @@ android {
 }
 
 dependencies {
-  testImplementation 'androidx.activity:activity-compose:1.6.1'
+  testImplementation libs.androidx.appcompat
```

Comment:
Brings in some compose but not all, `createLifecycleAwareWindowRecomposer` does not make it. These tests are not compose related, we have a separate compose lifecycle test, switching to appcompat since it covers these lifecycle tests.